js_modules/dagster-ui/packages/ui-core/src/instigation/InstigationTick.tsx

```diff
@@ -53,20 +53,19 @@ export const RunList = ({runIds}: {runIds: string[]}) => {
   );
 };
 
-export const FailedRunList = ({originRunIds}: {originRunIds?: string[]}) => {
+export const TargetedRunList = ({originRunIds}: {originRunIds?: string[]}) => {
   if (!originRunIds || !originRunIds.length) {
     return null;
   }
   return (
     <Group direction="column" spacing={16}>
       <Box padding={12} border={{side: 'bottom', color: Colors.textLighter()}}>
         <Body>
-          Failed Runs
-          <Tooltip content="Failed runs this tick reacted on and reported back to.">
+          Targeted Runs
```

Comment:
open to another verb here

Reply:
Reacted Runs?

Reply:
We use the word targeted elsewhere so I think it might be better to keep using it